jablib/src/test/java/org/jabref/model/database/BibDatabaseContextTest.java

```diff
@@ -185,6 +191,35 @@ void ofParsesValidBibtexStringCorrectly() throws Exception {
         assertEquals(List.of(expected), context.getDatabase().getEntries());
     }
 
+    @Test
+    void ofParsesValidBibtexStreamCorrectly() throws Exception {
+        String bibContent = """
+        @article{a,
```

Comment:
Lets try to keep test articles near realistic
```suggestion
        @article{Alice2023,
```
And corresponding change in the expected entry below